src/styles/block/calendar.scss

```diff
@@ -0,0 +1,29 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  padding: 10px;
+  width: getSize($cell-size, $cell-gap);
+
+  &__day {
+    @include square($cell-size);
+
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    font-family: Arial, sans-serif;
+    font-size: 30px;
+
+    border: 1px solid #000;
+    background-color: #eee;
+
+    transition: all 0.5s ease-in-out;
```

Comment:
don't use it like that, because it's the same as transition-duration: 0.5s
It's too hard for the browser
Check this
![image](https://user-images.githubusercontent.com/91411692/223074817-ce79a118-0b54-4537-b3fb-0e7364062d62.png)
also, create variable for transition Duration
```suggestion
    transition: transform $transitionDuration ease-in-out, background-color $transitionDuration ease-in-out;
```